fast64_internal/data/z64/xml/oot_enum_data.xml

```diff
@@ -683,4 +683,126 @@
+    <Item Key="sdc_chamber_of_the_sages" ID="SDC_CHAMBER_OF_THE_SAGES" Name="Chamber of the Sages (Kenjyanoma)" Index="32"/>
+    <Item Key="sdc_great_fairys_fountain" ID="SDC_GREAT_FAIRYS_FOUNTAIN" Name="Great Fairy Fountain" Index="33"/>
+    <Item Key="sdc_shooting_gallery" ID="SDC_SHOOTING_GALLERY" Name="Shooting Gallery (Syatekijyou)" Index="34"/>
+    <Item Key="sdc_castle_courtyard_guards" ID="SDC_CASTLE_COURTYARD_GUARDS" Name="Castle Hedge Maze (Day) (Hairal Niwa)" Index="35"/>
+    <Item Key="sdc_outside_ganons_castle" ID="SDC_OUTSIDE_GANONS_CASTLE" Name="Ganon's Castle Exterior (Ganon Tou)" Index="36"/>
+    <Item Key="sdc_ice_cavern" ID="SDC_ICE_CAVERN" Name="Ice Cavern (Ice Doukuto)" Index="37"/>
+    <Item Key="sdc_ganons_tower_collapse_exterior" ID="SDC_GANONS_TOWER_COLLAPSE_EXTERIOR" Name="Ganondorf's Death Scene (Tower Escape Exterior) (Ganon Final)" Index="38"/>
+    <Item Key="sdc_fairys_fountain" ID="SDC_FAIRYS_FOUNTAIN" Name="Fairy Fountain" Index="39"/>
+    <Item Key="sdc_thieves_hideout" ID="SDC_THIEVES_HIDEOUT" Name="Thieves' Hideout (Gerudoway)" Index="40"/>
+    <Item Key="sdc_bombchu_bowling_alley" ID="SDC_BOMBCHU_BOWLING_ALLEY" Name="Bombchu Bowling Alley (Bowling)" Index="41"/>
+    <Item Key="sdc_royal_familys_tomb" ID="SDC_ROYAL_FAMILYS_TOMB" Name="Royal Family's Tomb (Hakaana Ouke)" Index="42"/>
+    <Item Key="sdc_lakeside_laboratory" ID="SDC_LAKESIDE_LABORATORY" Name="Lakeside Laboratory (Hylia Labo)" Index="43"/>
+    <Item Key="sdc_lon_lon_buildings" ID="SDC_LON_LON_BUILDINGS" Name="Lon Lon Ranch House and Tower (Souko)" Index="44"/>
+    <Item Key="sdc_market_guard_house" ID="SDC_MARKET_GUARD_HOUSE" Name="Guard House (Miharigoya)" Index="45"/>
+    <Item Key="sdc_potion_shop_granny" ID="SDC_POTION_SHOP_GRANNY" Name="Granny's Potion Shop (Mahouya)" Index="46"/>
+    <Item Key="sdc_calm_water" ID="SDC_CALM_WATER" Name="Calm Water" Index="47"/>
+    <Item Key="sdc_grave_exit_light_shining" ID="SDC_GRAVE_EXIT_LIGHT_SHINING" Name="Grave Exit Light Shining" Index="48"/>
+    <Item Key="sdc_besitu" ID="SDC_BESITU" Name="Ganondorf Test Room (Besitu)" Index="49"/>
+    <Item Key="sdc_fishing_pond" ID="SDC_FISHING_POND" Name="Fishing Pond (Turibori)" Index="50"/>
+    <Item Key="sdc_ganons_tower_collapse_interior" ID="SDC_GANONS_TOWER_COLLAPSE_INTERIOR" Name="Ganon's Tower (Collapsing) (Ganon Sonogo)" Index="51"/>
+    <Item Key="sdc_inside_ganons_castle_collapse" ID="SDC_INSIDE_GANONS_CASTLE_COLLAPSE" Name="Inside Ganon's Castle (Collapsing) (Ganontika Sonogo)" Index="52"/>
+  </Enum>
+  <Enum Games="OoTMqDbg" Key="global_object" ID="GlobalObjects">
+    <Item Key="object_gameplay_field_keep" ID="OBJECT_GAMEPLAY_FIELD_KEEP" Name="Overworld" Index="1"/>
+    <Item Key="object_gameplay_dangeon_keep" ID="OBJECT_GAMEPLAY_DANGEON_KEEP" Name="Dungeon" Index="2"/>
+    <Item Key="object_invalid" ID="OBJECT_INVALID" Name="None" Index="0"/>
+  </Enum>
+  <Enum Games="MM" Key="cs_spline_rel" ID="CutsceneCamRelativeTo">
```

Comment:
MM stuff in an oot_ file?

Reply:
it's for the "enable MM features" checkbox